tests/integration/ssh/test_saltcheck.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import Salt Testing Libs
+from tests.support.case import SSHCase
+from tests.support.unit import skipIf
+
+# Import Salt Libs
+import salt.utils.platform
+
+
+@skipIf(salt.utils.platform.is_windows(), 'salt-ssh not available on Windows')
+class SSHSaltcheckTest(SSHCase):
+    '''
+    testing saltcheck with salt-ssh
+    '''
+    def test_saltcheck_run_test(self):
+        '''
+        test saltcheck.run_test with salt-ssh
+        '''
+        saltcheck_test = {"module_and_function": "test.echo",
+                        "assertion": "assertEqual",
+                        "expected-return": "Test Works",
+                        "args":["Test Works"] }
+        ret = self.run_function('saltcheck.run_test', test="saltcheck_test")

```

Comment:
This is obviously just passing the string "saltcheck_test", but this was set to attempt to debug the `None` test argument as described in the PR comments.